inspec.yml

```diff
@@ -13,7 +13,7 @@ inputs:
   description: Controls that are known to consistently have long run times can be disabled with this attribute
   # muted until https://github.com/inspec/inspec/issues/4799 is resolved
   #type: Boolean
```

Comment:
I think we turn Boolean back on I think they have fixed it - lets test

Reply:
So the current code already has this boolean set to true. This PR set it to false, if you are feeling that we should make it false we can just reject this PR. The reasoning behind setting this boolean to false is from the issue #101 :  
![image](https://user-images.githubusercontent.com/16725004/86818885-f0ebea00-c054-11ea-9341-a08ee1f07928.png)